Package.swift

```diff
@@ -227,7 +227,12 @@ let package = Package(
         ),
         .testTarget(
             name: "ClientRuntimeTests",
-            dependencies: ["ClientRuntime", "SmithyTestUtil", "SmithyStreams"],
+            dependencies: [
+                "ClientRuntime",
+                "SmithyTestUtil",
+                "SmithyStreams",
+                .product(name: "Logging", package: "swift-log"),
+            ],
```

Comment:
New logging test requires direct use of `swift-log` components, hence the test target dependency on `swift-log`